### PR TITLE
fix building error

### PR DESCRIPTION
**Pull request by lwintermelon**
`install.py all --web` doesn't work since added debug argument  detection to function `install_webapp` in `install.py`
https://github.com/sagemathinc/cocalc/blob/e30fe5e7a5684dcda58ed14d5b9f75e21cab8e2b/src/install.py#L84

### CODE REVIEW

File: src/install.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def install_webapp(*args):
         # update term.js
         cmd("cd webapp-lib/term; ./compile")
         wtype = 'debug' if args[0].debug else 'production'
```
Comment (williamstein) [inline src/install.py]: Thanks for your PR!  

It looks like there is still an args[0] in this line?

Comment (lwintermelon) [reply src/install.py]: I'm so careless...

Comment (williamstein) [reply src/install.py]: Pretty much everybody makes lots of mistakes when writing code no matter how long they try.   It's great to be writing code where one has the luxury of making mistakes (e.g., I hate writing code where i have to push it into production immediately and it must work or things get way worse -- that sucks).  Relax and enjoy.

**Comment by williamstein** [comment]
This doesn't seem to work:
```
~/smc/src$ git branch
...
* lwintermelon-patch-1
...
~/smc/src$ ./install.py all --web
Traceback (most recent call last):
  File "./install.py", line 166, in <module>
    main()
  File "./install.py", line 163, in main
    args.func(args)
  File "./install.py", line 160, in <lambda>
    parser_all.set_defaults(func = lambda args: install_all(compute=args.compute, web=args.web))
TypeError: install_all() got an unexpected keyword argument 'compute'
```

**Comment by lwintermelon** [comment]
Apologies, i should have tested it. it's ok now.

**Pull request closed without merging**